playground/frontend/playground_components_dev/lib/src/common_finders.dart

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:flutter_code_editor/flutter_code_editor.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:playground_components/playground_components.dart';
+
+extension CommonFindersExtension on CommonFinders {
+  Finder codeField() {
+    return byType(CodeField);
+  }
+
+  Finder graphTab() {
+    return widgetWithText(OutputTab, 'Graph');
```

Comment:
It's better to avoid find by text or at least use localization tools for this

Reply:
The benefit of finding by text is that if localization breaks we can catch it.
Also this is why we have this method. If we want to change the way to look up, it is a single location to change.

Reply:
Second thoughts, we will go with keys when we refactor the output tabs to use an enum:
- #22663